CONTRBUTING.md

```diff
@@ -0,0 +1,22 @@
+# How to contribute
+We love pull requests from everyone. Please take a look at the contributing
+guidelines below before making a pull request.
+
+## Getting started
+Make sure you have cloned the project under `$GOPATH/src` by using the `go get` command.
+
+## Formatting your code
+This project uses `gofmt` as standard code style. You can learn more about gofmt [here](https://blog.golang.org/go-fmt-your-code).
```

Comment:
I would remove the second sentence, and just link the blog post when you mention for the first time "gofmt"